assets/src/dashboard/components/scrollToTop/index.js

```diff
@@ -75,19 +75,23 @@ const DropUpArrowIcon = styled(ChevronLeft)`
 const ScrollToTop = () => {
   const [isVisible, setIsVisible] = useState(false);
   const {
-    actions: { scrollToTop, addSquishListener, removeSquishListener },
+    actions: { scrollToTop },
   } = useLayoutContext();
 
-  useEffect(() => {
-    const isVisibleFromProgress = (event) => {
-      setIsVisible(event.data.progress > 0);
-    };
+  const handleScroll = useCallback(() => {
+    if (window.scrollY > 0) {
+      setIsVisible(true);
+      return;
+    }
+    setIsVisible(false);
+  }, []);
 
-    addSquishListener(isVisibleFromProgress);
-    return () => {
-      removeSquishListener(isVisibleFromProgress);
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll);
```

Comment:
```suggestion
    window.addEventListener('scroll', handleScroll, { passive: true });
```

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#improving_scrolling_performance_with_passive_listeners